workinggroups/serverless.md

```diff
@@ -0,0 +1,39 @@
+# CNCF Serverless WG Proposal
+
+## TOC Sponsor
+
+Ken Owens
+
+## Objective
+
+Explore the intersection of cloud native and serverless technology.
+
+## Goals and Expected Outcomes
+
+* Produce a whitepaper
+* Produce a serverless landscape
+* Explore specifications for serverless to propose to the CNCF
```

Comment:
Explore specifications for serverless (e.g. events) to propose to the CNCF.

Reply:
s/Explore/Develop/

"Explore" isn't a goal.

Reply:
Is the WG evaluating existing specs or creating their own? Could "evaluate" be used instead of "explore"?